app/views/pages/our_mission.html.slim

```diff
@@ -0,0 +1,69 @@
+.mission-container
+  .grid-container
+    h3.title
+      ' Carta para
+      span.bg-highlight> todos
+      | nessa Pangeas
+    .grid-x.mission-text-container
+      .medium-1.cell.position-relative
+        = fa_icon 'quote-left 3x', class: 'icon-quote-left'
+      .medium-10.medium-centered.cell.mission-text
+        p
+          | Livre,
+          br
+          ' Eu sou. Livre porque
+          span.bg-highlight> sei questionar a liberdade:
+          ' Quem me
+          ' impede de comer melhor? O que permite que as doenças continuem
+          ' crescendo no mundo? Por que minhas emoções tem tanta força? O que
+          | cria o medo que me prende em casa? etc.
+        p
+          ' Mas não confundo liberdade com falta de responsabilidade ou com
+          ' fuga da realidade. Sei que
+          span.bg-highlight> posso escolher meu caminho
+          ' e que cada
+          ' escolha cria novos laços, para serem mantidos ou para serem
+          | desatados.
+        p
+          ' Por isso sei que para ser livre eu posso entender melhor como
+          ' funcionam minhas emoções. Posso descobrir quais realmente são os
+          ' alimentos e os movimentos que fazem meu corpo ter sua melhor forma.
+          ' Posso parar para respirar e sair do roteiro que o mundo inteiro
+          ' escreveu para eu interpretar. E, então,
+          span.bg-highlight>
+            ' observar melhor a causa (e não somente o sintoma) dos nossos
+            | problemas
+          ' de saúde, de
+          ' relacionamentos, políticos, econômicos, ambientais... Quem sabe até
+          ' entender o equilíbrio que a natureza descobriu em bilhões de anos e
+          ' finalmente poder parar de sugar/temer para começar somente a
+          | nutri-la.
+        p
+          ' O que tenho aqui, obviamente, precisa melhorar. Sempre precisar
+          ' melhorar. Porque quanto mais livre algo é, mais livre pode ser.
+          ' Porém, no meio de
+          span.bg-highlight> ideias novas e práticas ancestrais, já tenho
+          ' algumas pérolas para apresentar. Jóias duras o suficiente para
+          ' quebrar a identidade sem autoconsciência que nossa humanidade criou
+          | e belas o bastante para continuarmos querendo vê-las evoluir.
+        p
+          ' No mínimo, espero que as ideias que descubro em liberdade sejam
+          ' responsáveis por te ajudar a também
+          span.bg-highlight parar e respirar.
+      .medium-1.cell.position-relative
+        = fa_icon 'quote-right 3x', class: 'icon-quote-right'
+    .grid-x
```

Comment:
The part from here down is almost asking to become a partial (it appears in 3 pages already and there will be more).
But, the text on its header and the buttons isn't the same on all pages it appears. So I'm not sure if it's better to leave it hard-coded and just reuse the CSS classes or to use a partial.

Reply:
It seems a good idea to make it a partial, then, and pass `locals` to it with the custom texts.